backend/src/controllers/item.js

```diff
@@ -19,6 +19,14 @@ const ItemController = (itemModel, userModel, authService) => {
     const tags = body['tags']
     const sale_price = body['sale_price']
     const total_tickets = body['total_tickets']
+	
```

Comment:
nit: spacing is kind of oof here;  I think it is okay for the description and tags to be empty/null so maybe take those out of this

Reply:
Agreed^ frontend doesn't pass us tags right now

Reply:
Thanks I fixed that